docs/reference/commandline/service_create.md

```diff
@@ -177,6 +177,45 @@ $ docker service create \
   redis:3.0.6
 ```
 
+### Container networking
+The containers provisioned in docker swarm mode can be accessed in service discovery either via a Virtual IP (VIP) and routed through the docker swarm ingress overlay network. Or via a DNS round robbin (DNSRR). This configuration is set using the `--endpoint-mode [vip|dnsrr]` setting. Note that dnsrr is incompatible with `--published` ports.
+
+### Service discovery
+
+Discovering other service from within a container can be ac hived via DNS lookup, but you will get different results depending on whether you want the actual service IPs, or the VIP's.
```

Comment:
typo `ac hived` . This would be better stated: You can access a service from within a container _on the same overlay network_ using DNS lookup.